lib/ArrayStorage.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+var Cesium = require('cesium');
+
+var ComponentDatatype = Cesium.ComponentDatatype;
+
+module.exports = ArrayStorage;
```

Comment:
Do we want to add a dedicated spec for `ArrayStorage` in this PR? Or should that get roadmapped for now?